app/views/member/photos/index.html.erb

```diff
@@ -6,26 +6,26 @@
       <li class="list-item">
         <div class="info">
           <% link = @cur_node.parent ? { cid: @cur_node.parent } : cms_nodes_path %>
-          <%= link_to :parent_directory, link, class: "up" %>
+          <%= link_to t('ss.links.parent_directory'), link, class: "up" %>

```

Comment:
__[brakeman]__
* Unsafe parameter value in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->